### ISSUE TITLE
690. Employee Importance. Can't be solved because of the test case problem.

### ISSUE THREAD

**Issue by RakhmedovRS** [LeetCode-Feedback/LeetCode-Feedback#2662]
#### Your LeetCode username
rakhmedovrs

#### Category of the bug
- [x] Question
- [ ] Solution
- [ ] Language

#### Description of the bug
I solved the problem, but I can't submit it because of the test case problem. At first I thought that my solution contained the problem, then I checked it and resubmit and nothing changed. So I looked at my previous accepted solution and ran it, again nothing changed. Before submitting an issue I tool a solution from Solution's section and tried to run it problem was the same.
Every time I see this
![image](https://user-images.githubusercontent.com/25452084/110504889-a99b8900-810e-11eb-96ee-05750f02f0f1.png)

#### Code you used for Submit/Run operation

```
    public int getImportance(List<Employee> employees, int id)
    {
        Map<Integer, Employee> map = new HashMap<>();
        for (Employee employee : employees)
        {
            map.put(employee.id, employee);
        }
        
        return calcImportance(map.get(id), map, new HashMap<>());
    }

    private int calcImportance(Employee employee, Map<Integer, Employee> map, Map<Integer, Integer> memo)
    {
        if (memo.containsKey(employee.id))
        {
            return memo.get(employee.id);
        }

        int total = employee.importance;
        for (Integer subId : map.get(employee.id).subordinates)
        {
            total += calcImportance(map.get(subId), map, memo);
        }

        memo.put(employee.id, total);

        return total;
    }
```

#### Language used for code
Java

#### Expected behavior
Test case runs and give some comprehensive output

**Comment by RakhmedovRS** [comment]
@lc-kay I've checked it again, today everything works well. In case I face such or similar problem again what I should do? Should I create screenshots in developer console in a browser or something else?

**Comment by lc-kay** [comment]
@RakhmedovRS Yes, please provide as many details as possible in your reports so we can reproduce the issue to the best of our capabilities. Thank you for your time.
